plasmapy/analysis/nullpoint.py

```diff
@@ -0,0 +1,1357 @@
+"""Functionality to find and analyze 3D magnetic null points."""
+
+__all__ = [
+    "Point",
+    "NullPoint",
+    "null_point_find",
+    "trilinear_approx",
+]
+
+import numpy as np
+import warnings
+
+# Declare Constants & global variables
+_ATOL = 1e-10
+_MAX_DIVIDE = 10
+global _divide
+_divide = 0
```

Comment:
I'm wondering if it'd be helpful to increase the specificity of the name...perhaps something like `_recursion_level`?

Reply:
I agree. I changed the name of the variable as such.